tests/python/TestBoundingBox.py

```diff
@@ -0,0 +1,463 @@
+
+        bb_coords = {}
+        bb_coords["x"] = 100
+        bb_coords["y"] = 100
+        bb_coords["height"] = 180
+        bb_coords["width"] = 180
+
+        props = {}
+        props["name"] = "my_brain_bb"
+
+        bb = {}
+        bb["properties"] = props
+        bb["rectangle"] = bb_coords
+        bb["image"] = 12
+
+        bb_query = {}
+        bb_query["AddBoundingBox"] = bb
+
+        all_queries.append(bb_query)
+
+        response, res_arr = db.query(all_queries, [imgs_arr])
+
+        self.assertEqual(response[0]["AddImage"]["status"], 0)
+        self.assertEqual(response[1]["AddBoundingBox"]["status"], 0)
+
+    def test_findBoundingBoxesInImage(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        img_name = "my_brain_multiple"
+        imgprops = {}
+        imgprops["name"] = img_name
+        self.addBoundingBoxwithImage(db, number_of_inserts, imgprops)
+
+        all_queries = []
+
+        constraints = {}
+        constraints["name"] = ["==", img_name]
+        img = {}
+        img["constraints"] = constraints
+        img["_ref"] = 42
+        query = {}
+        query["FindImage"] = img
+        all_queries.append(query)
+
+        results = {}
+        results["list"] = ["_coordinates", "name"]
+
+        bb_params = {}
+        bb_params["image"] = 42
+        bb_params["results"] = results
+
+        query = {}
+        query["FindBoundingBox"] = bb_params
+
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["FindImage"]["status"], 0)
+        self.assertEqual(response[1]["FindBoundingBox"]["status"], 0)
+        self.assertEqual(response[1]["FindBoundingBox"]["returned"], number_of_inserts)
+
+        self.assertEqual(response[1]["FindBoundingBox"]["entities"][0]["_coordinates"]["x"], 10)
+        self.assertEqual(response[1]["FindBoundingBox"]["entities"][1]["_coordinates"]["x"], 0)
+        self.assertEqual(response[1]["FindBoundingBox"]["entities"][1]["name"], "my_brain_multiple_bb_0")
+
+
+    def test_findBoundingBoxByCoordinates(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        all_queries = []
+
+        rect_coords = {}
+        rect_coords["x"] = 0
+        rect_coords["y"] = 0
+        rect_coords["width"] = 100
+        rect_coords["height"] = 100
+
+        results = {}
+        results["list"] = ["name"]
+
+        bb_params = {}
+        bb_params["rectangle"] = rect_coords
+        bb_params["results"] = results
+
+        query = {}
+        query["FindBoundingBox"] = bb_params
+
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["FindBoundingBox"]["status"], 0)
+        # for i in range(0, len(response[0]["FindBoundingBox"]["entities"])):
+        #     print response[0]["FindBoundingBox"]["entities"][i]["name"]
+
+    def test_findBoundingBoxBlob(self):
```

Comment:
this tests is again not doing much, maybe check the contents of the response to add more value to this.